### DISCUSSION TITLE
[Question]: Detect motion when no object is detected

### DISCUSSION THREAD

**Discussion by gjz22** [blakeblackshear/frigate#10871] [Ask A Question] [answered]
### What is your question:

Is it possible for Frigate to detect when there is a motion event but there is no object that the current model is able to detect with a reasonable confidence? (Ideal further would be to require the motion event lasted an amount of time to ensure it wasn't just a light being turned on, for example)

The need for this is to be able to create a feedback loop over time with the model. The model starts with a certain number of labels, then "unknown" items are detected and new labels are created using those videos as training data and the cycle continues.

**Comment by NickM-27** [comment]
Yes, frigate detects motion and shows this on the motion MQTT topic. It also stores motion data for each recording segment that is saved.

**Reply by gjz22** [reply]
Right, but is there any way to get it to store motion data for all motion events that have no object detection labels of a given threshold?

**Reply by NickM-27** [reply, marked as answer]
Yes, it's stored in recordings table https://docs.frigate.video/configuration/record#reduced-storage-only-saving-video-when-motion-is-detected
